racer.el

```diff
@@ -460,6 +460,18 @@ Commands:
   (cons (get-text-property 0 'file arg)
         (get-text-property 0 'line arg)))
 
+(defun racer-get-bol-to-point ()
+  "Get text from start of line to point."
+  (let ((p (point)))
+    (save-excursion
+      (beginning-of-line)
+      (let ((bol (point)))
+        (buffer-substring-no-properties bol p)))))
+
+(defun racer-current-column ()
+  "Get the current column based on underlying character representation."
+  (string-width (racer-get-bol-to-point)))
```

Comment:
I think this is equivalent to simply `(length (buffer-substring (line-beginning-position) (point)))`. Do we need to use `string-width`? I've not tried Fira Symbols, but this seemed to do the right thing with prettify-symbols-mode.